src/EmbedIO/EmbedIO.csproj

```diff
@@ -21,6 +21,13 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>webserver websocket webapi http</PackageTags>
     <LangVersion>7.3</LangVersion>
+    <Configurations>Debug;Release;Signed</Configurations>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Signed'">
```

Comment:
`<PropertyGroup Condition="'$(Configuration)' == 'Release' And Exists('NugetSigning.pfx')">`
...and you no longer need a Signed configuration.